src/services/__tests__/redisSessionStorage.test.ts

```diff
@@ -124,31 +134,31 @@ describe("RedisSessionStorage#set", () => {
       undefined,
       new Error("hset error"),
       undefined,
-      left(new Error("Error setting the token")),
+      left(new Error("Error setting the token|hset error")),
       "should fail if Redis client returns false on saving the session and error saving the mapping"
     ],
     [
       undefined,
       undefined,
       undefined,
       undefined,
-      left(new Error("Error setting the token")),
+      left(new Error("Error setting the token|Error setting the token")),
```

Comment:
why two times?

Reply:
All the promises executed in a `Promise.all` on this [row](https://github.com/teamdigitale/io-backend/pull/432/files#diff-178a7f5864283f425f4e6d8d513710cfR79) always resolve. Previously only one type of error was returned if some of those promises returned a `left` value. You asked me to return an error with all the left occurrences [here](https://github.com/teamdigitale/io-backend/pull/432#discussion_r292425933)

Reply:
In this specific case one message refers the setting of `sessionToken`, the second refers the setting of `walletToken`

Reply:
perhaps then we should use two different terms to avoid confusion (i.e. `session token` and `wallet token`)

Reply:
fixed in 41a5323